marc_to_solr/lib/traject_config.rb

```diff
@@ -1189,6 +1189,7 @@
     context.output_hash['location_code_s'] = location_codes
     context.output_hash['location'] = Traject::TranslationMap.new("location_display").translate_array(location_codes)
     mapped_codes = Traject::TranslationMap.new("locations")
+    # It adds another owning library for the advanced search.
```

Comment:
I think it would be helpful to flesh this out a little bit more. Maybe "the holding_library is used with some locations to add an additional owning library, which is included in advanced search but not facets."